added a page with data indexers, included PARSIQ data provider in it

Ivan-Ivanitskiy opened the pull request:
Added a page with data indexers, included PARSIQ data provider in it.
![image](https://github.com/OffchainLabs/arbitrum-docs/assets/7903312/2af96614-ffce-4084-9c1b-121ca931667c)

symbolpunk commented:
Hi @Ivan-Ivanitskiy - thanks for the contribution. Would you mind reformatting this as a third-party doc following [the latest docs contribution guidelines](https://docs.arbitrum.io/for-devs/third-party-docs/contribute)?

If we find that our readers need general guidance on data indexers, we'll produce a core reference doc that includes PARSIQ. In the meantime, this could be a PARIQ-specific how-to, quickstart, or concept doc that helps readers understand how PARSIQ can help them as they build on Arbitrum. Eg:

```
📂 Third-party content
  📂 PARSIQ
    📄 (a how-to, quickstart, or concept doc here)
```

Ivan-Ivanitskiy commented:
Hey @symbolpunk ! Thank you for the feedback. I've reverted the changes and added PARSIQ Quickstart to third-party-docs, as recommended. Please take a look and have a great day!
![image](https://github.com/OffchainLabs/arbitrum-docs/assets/7903312/a319658f-7ea8-413c-8cba-a2a6b28baab7)

symbolpunk commented on arbitrum-docs/for-devs/third-party-docs/PARSIQ/parsiq.md:
```diff
@@ -0,0 +1,43 @@
+sidebar_label: "PARSIQ"
+---
+
+# PARSIQ Data API & SDK Quickstart
```
```suggestion
```

symbolpunk commented on arbitrum-docs/for-devs/third-party-docs/PARSIQ/parsiq.md:
```diff
@@ -0,0 +1,43 @@
+---
+title: "Get blockchain data with PARSIQ"
```
```suggestion
title: "Quickstart: PARSIQ (blockchain data API & SDK)"
```

symbolpunk commented on arbitrum-docs/for-devs/third-party-docs/PARSIQ/parsiq.md:
```diff
@@ -0,0 +1,43 @@
+
+PARSIQ API allows querying blockchain data such as transactions, token transfers, events, internal function calls, blocks, etc.
+
+PARSIQ SDK allows accumulationg, processing, and storing this data according to the developer-specified logic, as well as building your own API endpoints.
```
```suggestion
PARSIQ SDK allows accumulating, processing, and storing this data according to the developer-specified logic, as well as building your own API endpoints.
```

symbolpunk commented on arbitrum-docs/for-devs/third-party-docs/PARSIQ/parsiq.md:
```diff
@@ -0,0 +1,43 @@
+
+:::
+
+---
```
```suggestion
```

symbolpunk reviewed (changes requested):
Some minor nits @Ivan-Ivanitskiy, otherwise LGTM! Thanks for the contribution!
